release/make-release-artifacts.sh

```diff
@@ -152,15 +171,17 @@ git clean -dxf
 # Source release
 echo "Creating source release folder ${release_name}"
 set -x
-mkdir -p ${staging_dir}
-rsync -rtp --exclude src-release-tmp --exclude bin-release-tmp --exclude .git\* --exclude '**/*.[ejw]ar' --exclude docs/ . ${staging_dir}
```

Comment:
Have discussed this on Skype with Alex but for the record... the `--exclude '**/*.[ejw]ar'` **must** remain. No binaries in source releases. This was a blocker for our first release at Apache.